src/stories/payPalHostedFields/PayPalHostedFields.stories.tsx

```diff
@@ -72,51 +72,56 @@ function captureOrderUrl(orderId: string): string {
 const SubmitPayment = ({ customStyle }: { customStyle?: CSSProperties }) => {
     const [paying, setPaying] = useState(false);
     const cardHolderName = useRef<HTMLInputElement>(null);
-    const { cardFields: hostedField } = usePayPalHostedFields();
+    const hostedField = usePayPalHostedFields();
 
     const handleClick = () => {
-        if (hostedField) {
-            if (
-                Object.values(hostedField.getState().fields).some(
-                    (field) => !field.isValid
-                ) ||
-                !cardHolderName?.current?.value
-            ) {
-                return alert(
-                    "The payment form is invalid, please check it before execute the payment"
+        if (!hostedField?.cardFields) {
+            const childErrorMessage =
+                "Unable to find any child components in the <PayPalHostedFieldsProvider />";
+
+            action(ERROR)(childErrorMessage);
+            throw new Error(childErrorMessage);
+        }
+        const isFormInvalid =
+            Object.values(hostedField.cardFields.getState().fields).some(
+                (field) => !field.isValid
+            ) || !cardHolderName?.current?.value;
+
+        if (isFormInvalid) {
+            return alert(
+                "The payment form is invalid, please check it before execute the payment"
```

Comment:
I would prefer to simplify this error message:
```suggestion
                "The payment form is invalid"
```